style.css

```diff
@@ -100,3 +109,248 @@ p {
 .footer {
   margin-left: 10px;
 }
+
+.grid-container {
+  margin: 16px;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  row-gap: 100px;
+  background: #f7f7f9;
+}
+
+.template-wrapper {
+  margin: 16px;
+  border: 1px solid #f7f7f9;
+  border-radius: 10px;
+  background: white;
+  padding-bottom: 20px;
+}
+
+.wrapping {
+  margin: 10px;
+}
+
+.image-item {
+  margin-top: 10px;
+}
+
+.Tonic h1 {
+  font-size: 36px;
+  font-weight: 700;
+  color: #172b4d;
+  margin-bottom: 5px;
+}
+
+.Tonic-list ul {
+  display: flex;
+  justify-content: flex-start;
+  padding-left: 0;
+  margin-top: 3px;
+}
+
+.canopy {
+  list-style: none;
+  font-weight: 600;
+  font-size: 13px;
+  color: #344563;
+}
+
+.Tonic-list li {
+  margin: 0;
+  padding-right: 25px;
+  padding-top: 5px;
+}
+
+.canopy-list {
+  font-size: 13px;
+  font-weight: 600;
+  color: #7a869a;
+}
+
+.Tonic-content p {
+  font-weight: 400;
+  color: #344563;
+}
+
+.Tonic-link ul {
+  display: flex;
+  justify-content: flex-start;
+  padding-left: 0;
+  margin-top: 3px;
+}
+
+.Tonic-link li {
+  list-style: none;
+  padding-right: 10px;
+}
+
+.button-type {
+  padding: 10px 15px;
+  font-weight: 700;
+  font-size: 14px;
+  color: #6070ff;
+  border-radius: 10px;
+  background: #fff;
+  border: 1px solid #7f8cff;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: #fff;
+  text-decoration: underline;
+}
```

Comment:
- [Optional ] Please make sure the other button actions are working as well, as for now, this is the only one working.  and make sure you use the correct color for each action as precise by the previous reviewer